src/app/shared/pipes/greeting.pipe.ts

```diff
@@ -0,0 +1,14 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'greeting',
+  standalone: true, // If using Angular 15+
```

Comment:
Since we use Angular 19, the `standalone: true` option is optional (default). We can remove this line.